### ISSUE TITLE
About FTT-NAS GPU search problem...

### ISSUE THREAD

**Issue by BaronNashorMan** [walkerning/aw_nas#40]
Hello author, after I tried to complete all the environment installation and configuration according to your requirements, I first used the CPU to search for the structure and found that ENAS and FTT-NAS are no problems, but when I use GPU to search, ENAS is no problem. But FTT-NAS has a problem, I don’t know how to solve this problem now...
....................................................
The following is an error message......
...................................................
fault_injection.py", line 244, in get_reward    perfs = self.get_perfs(inputs, outputs, targets, cand_net)
fault_injection.py", line 254, in get_perfs outputs_f = cand_net.forward_one_step_callback(inputs, callback=self.inject) super_net.py", line 175, in forward_one_step_callback callback(context.last_state, context)
fault_injection.py", line 337, in inject context.last_state = self.injector.inject(state, n_mac=n_mac)
fault_injection.py", line 180, in inject  return eval("self.inject_" + self.mode)(out, **kwargs)
fault_injection.py", line 159, in inject_fixed size=size_)])
prod() received an invalid combination of arguments - got (out=NoneType, axis=NoneType, )

**Comment by walkerning** [comment]
Hi, what cmd-line command did you run? I check our ftt-nas doc, and found some errors in the doc. And I've updated the doc now. Can you try to follow current doc in https://github.com/walkerning/aw_nas/tree/master/examples/research/ftt-nas , and see whether it works now?

**Comment by BaronNashorMan** [comment]
Thank you for taking the time out of your busy schedule to respond to my questions.
I imitated the ENAS search command line on your project homepage to perform FTT-NAS search. The command line I ran was:

`awnas search examples/research/ftt-nas/mibb.yaml --gpu 3 --save-every 10 --train-dir ftt_nas_search`

You have updated the project now. I will follow the instructions in your project document to proceed. 
I believe that following your guidance, this problem will be resolved soon. 
Thank you very much!
